docs/content/preview/api/ysql/the-sql-language/statements/dml_fetch.md

```diff
@@ -53,20 +53,23 @@ A _cursor_ represents the current position in its result set. After declaring a
 
 - The `FETCH :n` and `FETCH FORWARD :n` variants fetch exactly _:n_ rows forwards from and including the row after the current position when this many rows are available and otherwise just as many as there are to fetch analogously to how `FETCH FORWARD ALL` behaves.
-- The `FETCH BACKWARD :n` variant fetches exactly _:n_ rows backwards from and including the row before the current position when this many rows are available and otherwise just as many as there are to fetch analogously to how `FETCH BACKWARD ALL` behaves.
 
-- The `FETCH ABSOLUTE :n` variant fetches the single row at exactly the indicated absolute position. The `FETCH RELATIVE :n` variant fetches the single row at exactly the indicated relative position (_:n_ can be negative) to the current row. For both `FETCH ABSOLUTE :n` and `FETCH RELATIVE :n`, the requested row might lie before the first row or after the last row. The outcome here is the same as it is when executing other `FETCH` variants cause the current position to fall outside the range from the first through the last row in the cursor's result set. Notice that _:n_ can be negative for both the `ABSOLUTE` and the `RELATIVE` variants.
+- The `FETCH BACKWARD :n` variant fetches exactly _:n_ rows backwards from and including the row before the current position when this many rows are available and otherwise just as many as there are to fetch analogously to how `FETCH BACKWARD ALL` behaves\*.
 
-- Each of the `FETCH FIRST` and `FETCH LAST` variants fetches, respectively, the first row or the last row. The meanings are therefore insensitive to the current cursor position, and each can be repeated time and again and will always produce the same result. 
+- The `FETCH ABSOLUTE :n` variant fetches the single row at exactly the indicated absolute position. The `FETCH RELATIVE :n` variant fetches the single row at exactly the indicated relative position (_:n_ can be negative) to the current row. For both `FETCH ABSOLUTE :n` and `FETCH RELATIVE :n`, the requested row might lie before the first row or after the last row. The outcome here is the same as it is when executing other `FETCH` variants cause the current position to fall outside the range from the first through the last row in the cursor's result set. Notice that _:n_ can be negative for both the `ABSOLUTE` and the `RELATIVE` variants\*.
 
-Notice that the three variants ,`FETCH FORWARD 0`, `FETCH BACKWARD 0`, and `FETCH RELATIVE 0`, all mean the same as each other.
+- Each of the `FETCH FIRST` and `FETCH LAST` variants fetches, respectively, the first row or the last row. The meanings are therefore insensitive to the current cursor position, and each can be repeated time and again and will always produce the same result\*.
+
+Notice that the three variants ,`FETCH FORWARD 0`, `FETCH BACKWARD 0`, and `FETCH RELATIVE 0`, all mean the same as each other\*.
+
+**\[\*\]** Subject to what [Issue #6514](https://github.com/yugabyte/yugabyte-db/issues/6514) reports.
```

Comment:
why in this case link to the github issue, rather than the warning section on the cursors page as elsewhere

Reply:
Good idea. I'll make that change.